dioptas/widgets/plot_widgets/ColormapPopup.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+# Dioptas - GUI program for fast processing of 2D X-ray diffraction data
+# Principal author: Clemens Prescher (clemens.prescher@gmail.com)
+# Copyright (C) 2014-2019 GSECARS, University of Chicago, USA
+# Copyright (C) 2015-2018 Institute for Geology and Mineralogy, University of Cologne, Germany
+# Copyright (C) 2019-2020 DESY, Hamburg, Germany
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+import pathlib
+
+from qtpy import QtGui, QtCore, QtWidgets
+import pyqtgraph.graphicsItems.GradientEditorItem
+
+from ... import style_path
+
+
+class ColormapPopup(QtWidgets.QFrame):
+    """Dialog providing control over the currently used colormap"""
+
+    sigCurrentGradientChanged = QtCore.Signal(dict)
+    """Signal emitted when the colormap gradient has changed"""
+
+    sigRangeChanged = QtCore.Signal(float, float)
+    """Signal emitted when the data range has changed"""
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.setWindowTitle("Colormap configuration")
+        self.setWindowFlags(QtCore.Qt.Popup)
+        self.setAttribute(QtCore.Qt.WA_DeleteOnClose)
+        self.setStyleSheet(
+            pathlib.Path(style_path, "stylesheet.qss").read_text()
+        )
+        self.setFrameStyle(QtWidgets.QFrame.Panel | QtWidgets.QFrame.Raised)
+        self.setLineWidth(2)
+
+        frameLayout = QtWidgets.QVBoxLayout(self)
+        frameLayout.setContentsMargins(3, 3, 3, 3)
+        groupbox = QtWidgets.QGroupBox("Colormap", self)
```

Comment:
I added a `QGroupBox` to make the colormap options more visible on top of the underlying widgets